src/Raven.Studio/typescript/viewmodels/database/status/storageReport.ts

```diff
@@ -172,28 +174,74 @@ class storageReport extends viewModelBase {
         };
         return preAllocatedBuffers;
     }
+    
+    private groupChildren(items: storageReportItem[]): storageReportItem[] {
+        
+        if (items.length > storageReport.maxChildrenToShow) {
+            let clonedItems = _.cloneDeep(items);
```

Comment:
why you deep clone here? Can we avoid that?

Reply:
changed to `_.clone()`

Reply:
Can we avoid clone at all?

Reply:
yes - removed